Source/Private/ASDisplayNodeInternal.h

```diff
@@ -173,6 +173,10 @@ FOUNDATION_EXPORT NSString * const ASRenderingEngineDidDisplayNodesScheduledBefo
 
   // keeps track of nodes/subnodes that have not finished display, used with placeholders
   ASWeakSet *_pendingDisplayNodes;
+  
+  CGFloat _cornerRadius;
+  ASCornerRoundingType _cornerRoundingType;
+  CALayer *_clipCornerLayers[4];
```

Comment:
Hi Scott! How are these ObjC objects retained in this C array? Is there ARC magic I'm clearly not aware of? -- Greg